frontend/src/utils/ApiWrapper.js

```diff
@@ -914,7 +913,9 @@ export const updateDocumentStatus = (userID, id, status) => {
       }
     })
     .then(response => {
-      createMessage(userID, false, true, id)
+      reason

```

Comment:
Do you still need to check for reason existing here if you also check it in line 1092

Reply:
+1